### PR TITLE
small parentsUntil fixes

### PR DESCRIPTION
**Pull request by 5saviahv**
- parentsUntil selector accepts multiple results.
- parentsUntil filter didnt filter

### CODE REVIEW

**Comment by fb55** [comment]
@5saviahv This is a great find, any reason you closed the PR?

**Comment by 5saviahv** [comment]
I was panicking when one of my tests fail. Turns out I had typo in tests and it wasn't even pushed github.

File: lib/api/traversing.js [hunk 1]
```diff
@@ -152,11 +152,11 @@ exports.parentsUntil = function (selector, filter) {
   var untilNodes;
 
   if (typeof selector === 'string') {
-    untilNode = select.select(
+    untilNodes = select.select(
```
Comment (fb55) [inline lib/api/traversing.js:155]: Shouldn't this be a `filter` as well?

Comment (5saviahv) [reply lib/api/traversing.js:155]: now when you say it ... yes it should.

**Comment by fb55** [comment]
Nice stuff!

**Pull request merged**